src/popover/index.js

```diff
@@ -78,7 +78,7 @@ class Popover extends Component {
   }
 
   _getBackDropElement = () => {
-    if (!this.props.onClickOut) return null;
+    if (!this.props.onClickOut || !this.props.isOpen) return null;
```

Comment:
How? If no onClickOut is provided, the condition passes (!this.props.onClickOut) and it returns null. The only time this condition will pass to render the overlay is when there is a click handler specified and it's open. Arguably i could reorient the question but the logic is sound.